gds/src/main/scala/edu/gemini/aspen/gds/fits/FitsFileTransferrer.scala

```diff
@@ -208,5 +208,5 @@ object FitsFileTransferrer {
       )
         .through(Files[F].writeAll(output))
         .compile
-        .drain
+        .count
```

Comment:
Does this need to be replaced with a tap and a Ref as in the discussion in the Arman slack channel?

Reply:
Yes it does, I want to log the file size